input-files/emsl_header.json

```diff
@@ -79,7 +79,7 @@
     },
     "Storage Condition": {
         "1": "Detail how your sample was stored",
-        "2": "{YYYY-MM-DD}",
+        "2": "enumeration",
```

Comment:
I looked at the linked issue just now – did @mslarae13 mean for this to be changed to `{text}`?